app/src/main/java/org/openobservatory/ooniprobe/model/database/Url.java

```diff
@@ -38,6 +48,15 @@ public static Url getUrl(String input) {
 		return SQLite.select().from(Url.class).where(Url_Table.url.eq(input)).querySingle();
 	}
 
+	public static List<Url> getExistingUrls(Collection<String> urls) {
+		return SQLite.select().from(Url.class).where(Url_Table.url.in(urls)).queryList();
+	}
+
+	public static void saveAll(Collection<Url> urls) {
```

Comment:
```suggestion
	private static void saveAll(Collection<Url> urls) {
```

This function does not seem to be used publicly